examples/bri-3/src/bri/state/agents/state.agent.ts

```diff
@@ -20,6 +24,7 @@ import { Witness } from '../../zeroKnowledgeProof/models/witness';
 @Injectable()
 export class StateAgent {
   constructor(
+    @InjectMapper() private mapper: Mapper,
     private bpiAccountStorageAgent: BpiAccountStorageAgent,
     private merkleTreetStorageAgent: MerkleTreeStorageAgent,
```

Comment:
merkleTreetStorageAgent (Mis-spelt) --> merkleTreeStorageAgent

Reply:
Good catch. Its because its a Treat + tree = Treet. We will change in one of the next PRs.